release.nix

```diff
@@ -0,0 +1,15 @@
+let
+  overlay = import ./pkgs;
+  nixpkgs = import (import ./nixpkgs.nix) {
+    config.allowUnfree = true;
```

Comment:
I don't think it's a good idea to set `allowUnfree` in an open source project.

Reply:
Depends on what you put in the overlay, I guess.

Reply:
No, the point is that you make that decision for everything in the nixpkgs pin and this option exists in order to make that an explicit choice. Even if the whole closure was useless without `allowUnfree = true`, users should still be required to pass that explicitly in order to acknowledge they're willing to use unfree software.